### PR TITLE
Added 500.30 Key Vault note - a common issue

### PR DESCRIPTION
**Pull request by samsmithnz**
This 500.30 issue seems to pop up a lot, but doesn't show much information. When using Key Vault, this seems to be a common developer error.

### CODE REVIEW

File: aspnetcore/test/troubleshoot-azure-iis.md [hunk 1]
```diff
@@ -113,6 +113,8 @@ The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) attempts to s
 
 A common failure condition is the app is misconfigured due to targeting a version of the ASP.NET Core shared framework that isn't present. Check which versions of the ASP.NET Core shared framework are installed on the target machine.
 
+If using Azure Key Vault, another common failure condition is lack of permissions to the Key Vault. Check the access policies in the targeted Key Vault to ensure the website user has access to read secrets.
```
Comment (Rick-Anderson) [inline aspnetcore/test/troubleshoot-azure-iis.md]: A common failure condition with Azure Key Vault is lack of permissions to the Key Vault. Check the access policies in the targeted Key Vault to ensure the website user has access to read secrets.

**Comment by guardrex** [comment]
Hold on a sec.

**Comment by guardrex** [comment]
That's not accurate advice.

**Comment by guardrex** [comment]
:smile: ... I guess "precise" would be a better way to say it. Let me confirm ... I'll get right back to you.

File: aspnetcore/test/troubleshoot-azure-iis.md [hunk 2]
```diff
@@ -111,7 +111,10 @@ The worker process fails. The app doesn't start.
 
 The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) attempts to start the .NET Core CLR in-process, but it fails to start. The cause of a process startup failure can usually be determined from entries in the Application Event Log and the ASP.NET Core Module stdout log.
 
-A common failure condition is the app is misconfigured due to targeting a version of the ASP.NET Core shared framework that isn't present. Check which versions of the ASP.NET Core shared framework are installed on the target machine.
+Common failure conditions:
+
+* The app is misconfigured due to targeting a version of the ASP.NET Core shared framework that isn't present. Check which versions of the ASP.NET Core shared framework are installed on the target machine.
+* Using Azure Key Vault, lack of permissions to the Key Vault. Check the access policies in the targeted Key Vault to ensure the website user has access to read secrets.
```
Comment (guardrex) [inline aspnetcore/test/troubleshoot-azure-iis.md]:
```suggestion
* Using Azure Key Vault, lack of permissions to the Key Vault. Check the access policies in the targeted Key Vault to ensure that **Get** and **List** secret permissions are granted.
```

... something like that. I don't recommend that it say "website user" (e.g., Managed Identities). The exact permissions are "Get" and "List," and those are "secrets" permissions settings.

**Pull request merged**